asv_bench/benchmarks/scalability/scalability_benchmarks.py

```diff
@@ -13,7 +13,11 @@
 
 import modin.pandas as pd
 from modin.pandas.utils import from_pandas
-from modin.utils import to_pandas
+
+try:
+    from modin.utils import to_pandas
+except ImportError:
+    from modin.pandas.utils import to_pandas
```

Comment:
why?

Reply:
This is a compatibility question - to be able to run benchmarks on old Modin commits.

Reply:
@anmyachev Would you add a short comment here explaining?

Reply:
added